lib/routes/account.js

```diff
@@ -425,6 +432,15 @@ module.exports = function (
             .then(
               function (result) {
                 device = result
+                if (operation === 'createDevice') {
+                  log.event('device', {
+                    action: 'create',
```

Comment:
I would prefer something like `device:create` and `device:delete`, but every other event currently is a single top level word. The oauth server listens for `delete` events, which actually means account deletion.